roles/ovirt-repositories/tasks/rpm.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Backup repositories
+  include: backup-repos.yml
+
+- name: Install rpm repository package
+  yum:
+    name: "{{ ovirt_rpm_repo }}"
```

Comment:
You can use ' ovirt_rpm_repo|mandatory' to enforce the var to be passed.

Reply:
nice